ElectronClient/gui/MainScreen.jsx

```diff
@@ -140,6 +141,7 @@ class MainScreenComponent extends React.Component {
 								this.props.dispatch({
 									type: 'FOLDER_SELECT',
 									id: folder.id,
+									historyNoteAction: HistoryHelper.getNoteAction(this.props.selectedNoteIds, this.props.notes),
```

Comment:
historyNoteAction contains the **id** and **parent_id** of the note we are navigating away from. 
Maybe it should also be renamed? Along with the function getNoteAction. Suggestions?